### PR TITLE
prepping refactoring of image build process

### PR DESCRIPTION
**Pull request by dgraeber**
### Description:

layering dockerfiles and restructuring / removing unnecessary artifacts

### Issue Reference URL

https://github.com/awslabs/aws-orbit-workbench/issues/1183

----------
### Do not change content below. Mark applicable check boxes only.

Thank you for submitting a contribution to AWS Orbit Workbench.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a ISSUE associated with this PR?
- [ ] Does your PR title start with ISSUE-XXXX where XXXX is the issue number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [ ] Has your PR been rebased against the latest commit within the target branch (typically 'main')?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under Apache License 2.0?

### CODE REVIEW

File: cli/aws_orbit/commands/deploy.py [hunk 1]
```diff
@@ -318,6 +318,84 @@ def deploy_env(
         msg_ctx.progress(100)
 
 
+def deploy_base_images(
```
Comment (chamcca) [inline cli/aws_orbit/commands/deploy.py:321]: is this needed right now? it's still using the old `remote.run` and `generate_bundle` functions. all of that is replaced by the toolkit

File: images/jupyter-user/Dockerfile [hunk 2]
```diff
@@ -1,9 +1,7 @@
-ARG BASE_IMAGE=public.ecr.aws/v3o4w1g6/aws-orbit-workbench/jupyter/base-notebook:python-3.8.8
-FROM $BASE_IMAGE
 
-######################################
-# System
-######################################
+ARG BASE_IMAGE=public.ecr.aws/v3o4w1g6/aws-orbit-workbench/jupyter/base-notebook:python-3.8.8
+#ARG BASE_IMAGE=jupyter/base-notebook:python-3.8.8
```
Comment (chamcca) [inline images/jupyter-user/Dockerfile:3]: remove commented arg?

File: cli/aws_orbit/commands/deploy.py (hunk 1 above)
Comment (dgraeber) [reply cli/aws_orbit/commands/deploy.py:321]: I will remediate

File: images/jupyter-user/Dockerfile (hunk 2 above)
Comment (dgraeber) [reply images/jupyter-user/Dockerfile:3]: yes...will do

File: images/utility-data/Dockerfile [hunk 3]
```diff
@@ -1,23 +1,23 @@
-FROM public.ecr.aws/v3o4w1g6/aws-orbit-workbench/k8s-utilities:1.0.0
+FROM public.ecr.aws/v3o4w1g6/aws-orbit-workbench/k8s-utilities:1.0.0 as utilbase
 
 RUN pip install boto3
 
-RUN mkdir -p /opt/orbit/scripts
+RUN mkdir -p /opt/orbit/scripts &&
```
Comment (chamcca) [inline images/utility-data/Dockerfile:5]: trailing &&

Comment (dgraeber) [reply images/utility-data/Dockerfile:5]: AGH!

**Review by chamcca — APPROVED** [review]

**Review by srinivasreddych — APPROVED** [review]

**Pull request merged**